src/main/java/com/metamx/metrics/SysMonitor.java

```diff
@@ -382,10 +394,21 @@ public void emit(ServiceEmitter emitter)
               }
               if (netstat != null) {
                 final Map<String, Long> stats = diff.to(
-                    name, ImmutableMap.of(
-                    "sys/net/read/size", netstat.getRxBytes(),
-                    "sys/net/write/size", netstat.getTxBytes()
-                )
+                    name, ImmutableMap.<String, Long>builder()
+                      .put("sys/net/read/size", netstat.getRxBytes())
+                      .put("sys/net/read/packets", netstat.getRxPackets())
+                      .put("sys/net/read/errors", netstat.getRxErrors())
+                      .put("sys/net/read/dropped", netstat.getRxDropped())
+                      .put("sys/net/read/overruns", netstat.getRxOverruns())
+                      .put("sys/net/read/frame", netstat.getRxFrame())
+                      .put("sys/net/write/size", netstat.getTxBytes())
+                      .put("sys/net/write/packets", netstat.getTxPackets())
+                      .put("sys/net/write/errors", netstat.getTxErrors())
+                      .put("sys/net/write/dropped", netstat.getTxDropped())
+                      .put("sys/net/write/collisions", netstat.getTxCollisions())
+                      .put("sys/net/write/overruns", netstat.getTxOverruns())
+                      .put("sys/net/write/carrier", netstat.getTxCarrier())
```

Comment:
@xvrl, have lost your comment

(http://www.linuxquestions.org/questions/linux-networking-3/what-is-carrier-from-the-ifconfig-output-57333/#post293740)

Not sure, that it is a useful metric